yaml/water.yaml

```diff
@@ -18,6 +18,27 @@ globals:
           - when:
               intermittent: 'yes'
             then: true
+  - &water_polygon_min_zoom
+    lookup:
+      key: { col: way_area }
+      op: '>='
+      table:
+        # limit show smaller landuse areas to higher zooms
+        - [ 0, 40000000000 ]
+        - [ 2, 20000000000 ]
+        # some weird natural earth scale set transition
+        - [ 3, 80000000000 ]
+        - [ 4,  5000000000 ]
+        - [ 5,   700000000 ]
+        - [ 6,   500000000 ]
+        - [ 7,   160000000 ]
+        - [ 8,    20000000 ]
+        - [ 9,     5000000 ]
+        - [ 10,    1000000 ]
+        - [ 11,     200000 ]
+        - [ 12,      50000 ]
+        - [ 13,      20000 ]
+      default: 14
```

Comment:
This seems to now show tiny lakes and swimming pools at zoom 14 instead of zoom 16 max? Although in practice browsing around Cupertino (link below) most swimming pools were getting zoom 14.xxx incorrectly (and even showing before the surrounding house footprints!)

Suggest adding zoom 14, 15, 16, and default 17 instead:

```
        - [ 14,      2000 ]
        - [ 15,      1000 ]
        - [ 16,      400 ]
      default: 17
```

I've been browsing around Cupertino to gain some of these numbers:

- https://tangrams.github.io/walkabout-style/#17/37.32740/-122.04532

Reply:
Well spotted, thanks! Fixed in 91c05f0.